recipes/qiskit-aer/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "qiskit-aer" %}
+{% set version = "0.11.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Qiskit/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: d18aefd09971bfe2018dc78623ddf8a25bc0660f4fb0987a82c8e2a677bffc4e
+  patches:
+    # This patch is used to prevent scikit-build from trying to pip install cmake.
+    # It can be dropped when a version including
+    # https://github.com/Qiskit/qiskit-aer/pull/1693 is released, likely
+    # version 0.12.0.
+    - 0001-Drop-setup_requires.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  # Skip Windows build until qiskit-terra has a Windows build
+  # Skip macOS build until https://github.com/Qiskit/qiskit-aer/issues/1689 is addressed
+  skip: True  # [win or osx]
+  number: 0
+  script_env:
+    - DISABLE_DEPENDENCY_INSTALL=ON
+    - DISABLE_CONAN=ON
+    - SKBUILD_CONFIGURE_OPTIONS="-GNinja"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake !=3.17,>=3.8
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx]
+    - ninja
+  host:
+    - libblas
+    - liblapack
+    - muparserx =4.0.8
+    - nlohmann_json =3.1.2
+    - numpy
```

Comment:
Q: Is NumPy header used at build time?

Reply:
I was guessing so since it is in the [`build-system.requires of qiskit-aer's pyproject.toml](https://github.com/Qiskit/qiskit-aer/blob/f22d61208b18c948dbc257dbd863fbeddf060b28/pyproject.toml#L10). The main simulator is in C++ with Python bindings, so I was thinking the numpy headers would be needed for passing state vector and density matrix results across that interface. I don't see any direct references to numpy headers in the code but I am not that familiar with pybind11 which might provide a layer between numpy and C++? I think the conversion is partially specified [here](https://github.com/Qiskit/qiskit-aer/blob/5d30c4ea1351417ea8d386c5f228c9afaeb15f3d/src/framework/pybind_basics.hpp#L38-L42).